R/functions.R

```diff
@@ -73,32 +79,33 @@ clean_covariates <- function(md, factors, continuous) {
 #'
 #'This function produces boxplots from the variables provided.
 #'
-#'@inheritParams md
-#'@param vars A vector of variables to visualize
+#'@inheritParams coerce_factors
+#'@param include_vars A vector of variables to visualize
+#'@param x_var Variable to plot on the x-axis.
+#'@importFrom rlang .data
 #'
 #'@export
 #'@return
```

Comment:
This line is causing a NOTE. `@return` is for documenting what the function returns, so to prevent the NOTE you could either remove this line or add a description of what is returned by the function.